wls-gui-wahllokalsystem/src/composables/basisdaten/ungueltigeWahlscheineMapper.ts

```diff
@@ -0,0 +1,36 @@
+import type { UngueltigerWahlschein } from "@/types/wahlbezirk/UngueltigerWahlschein.ts";
+
+const NEW_LINE = "\n";
+const VALUE_DELIMITER = ";";
+
+export function useUngueltigeWahlscheineMapper() {
+  const removeSpacesAndQuotationsCallback = (entry: string) =>
+    entry.replace(/[\s"]+/g, "");
+
+  function toModel(
+    ungueltigeWahlscheineCSVString: string
+  ): UngueltigerWahlschein[] {
+    return ungueltigeWahlscheineCSVString
+      .split(NEW_LINE)
+      .map(removeSpacesAndQuotationsCallback)
+      .map((line) => csvLineToModel(line))
+      .filter((ungueltigerWahlschein) => ungueltigerWahlschein !== undefined);
+  }
+
+  function csvLineToModel(
```

Comment:
immer noch ne interne funktion
```suggestion
  function _csvLineToModel(
```

Reply:
fixed in 10795859204bf90bc8350634fe7744107a97a204